README.md

```diff
@@ -1,53 +1,82 @@
 # ASF On AWS Open Data
 
-This is a repository to keep a record of and track how the Alaska Satellite Facility (ASF) Tools Team provides and manages data on AWS OpenData.
+This is a repository to keep a record of and track how the Alaska Satellite Facility (ASF) Tools Team 
```

Comment:
```suggestion
This is a repository to keep a record of and track how the Alaska Satellite Facility (ASF) 
```
I think we should drop the team specifier since this repo is now broader, or change it to Tools and SciDev.